docs/contents/components/forms/dropzone/index.mdx

```diff
@@ -175,10 +191,24 @@ To control the display based on whether the file is accepted or rejected, use `D
     </DropzoneIdle>
 
     <VStack gap="2xs">
-      <Text fontSize="xl">Drag images here or click to select files</Text>
-      <Text fontSize="sm">
-        Attach as many files as you like, each file should not exceed 3mb
-      </Text>
+      <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.

Reply:
Fixed.